### PR TITLE
DOCS-2911: SCALE Rsync Task Article and Video

### PR DESCRIPTION
**Pull request by micjohnson777**
Publish new SCALE resync task article and video, update CORE resync task article style, fix grammar and clarify the process.

Thanks for contributing to TrueNAS documentation! By opening a Pull Request, you're acknowledging that your changes will be distributed under the [Creative Commons 4.0](https://creativecommons.org/licenses/by-nc-sa/4.0/) license.

### CODE REVIEW

File: content/CORE/Tasks/Rsync.md [hunk 1]
```diff
@@ -5,83 +5,83 @@ weight: 30
 
 {{< toc >}}
 
-Data often needs to be copied to another system for backup or when migrating to a new system.
+You often need to copy data to another system for backup or when migrating to a new system.
 A fast and secure way of doing this is by using [rsync](https://rsync.samba.org/).
-These instructions assume that TrueNAS systems are being used for both sides of the rsync task: *host* and *remote*.
+These instructions assume that both sides of the rsync task, host and remote, use a TrueNAS systems.
```
Comment (tonyriv3) [inline content/CORE/Tasks/Rsync.md:10]: These instructions assume that both sides of the rsync task, host and remote, use TrueNAS systems.

File: content/CORE/Tasks/Rsync.md [hunk 2]
```diff
@@ -5,83 +5,83 @@ weight: 30
...
 ## Basic Requirements
 
-Rysnc requires a [dataset]({{< relref "Datasets.md" >}}) with the needed data on the *host* or *remote* system.
-Rsync provides the ability to either *push* or *pull* data.
-When using rsync to *push*, data is copied from a *host* system to a *remote* system.
-When using rsync to *pull*, data is pulled from a *remote* system and put on the *host* system.
+Rysnc requires a [dataset]({{< relref "Datasets.md" >}}) with the needed data on the host or remote system.
+Rsync provides the ability to either push or pull data.
+When using the **Rsync Tasks** function to push, data is copied from a host system to a remote system.
+When using **Rsync Tasks** function to pull, data is pulled from a remote system and put on the host system.
```
Comment (tonyriv3) [inline content/CORE/Tasks/Rsync.md:17]: When using the **Rsync Tasks** function to pull, it pulls data from a remote system and puts it on the host system.

File: content/SCALE/DataProtection/RsyncTasksSCALE.md [hunk 3]
```diff
@@ -0,0 +1,211 @@
...
+![SystemServicesEditRsyncConfigureTab](/images/SCALE/SystemServicesEditRsyncConfigureTab.png "System Services Edit Rsync Configure Tab")
+
+{{< include file="static/includes/Reference/ServicesRsyncConfigureFields.md.part" markdown="true" >}}
+
+Use the default settings unless a specific change is required.
+Remember to click **Save** after changing any settings.
+{{< /tab >}}
+{{< tab "Rsync Module" >}}
+
+All created modules are listed on the **Rsync Module** tab.
```
Comment (tonyriv3) [inline content/SCALE/DataProtection/RsyncTasksSCALE.md:190]: The **Rsync Module** tab lists all created modules.

**Review by Mrt134 — APPROVED** [review]
Looks great, thanks!

**Pull request merged**